dev/com.ibm.ws.cdi.third.party_fat/fat/src/com/ibm/ws/cdi12/suite/FATSuite.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2018, 2020 IBM Corporation and others.
+ * Copyright (c) 2018, 2021 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
```

Comment:
The copyright header has been changed but there are no other changes to the class. Should the `RepeatTests` rule have been updated?

Reply:
Most likely.